modules/subscriber/Dockerfile.gobuild

```diff
@@ -2,5 +2,6 @@ FROM golang:alpine
 
 COPY main.go /opt/
 WORKDIR /opt
+RUN go env -w GO111MODULE=off
```

Comment:
Since `go get` is deprecated and it will be not available in Go 1.18 at all, I think it would be better to change base image to `golang:1.15-alpine3.14` to keep it stable and avoid the same issue once `golang:alpine` is updated to 1.18.